aci_create_with_acr.yml

```diff
@@ -11,17 +11,24 @@
 # ===========
 # - git
 # - Docker
+- hosts: localhost
+  tasks:
+    - name: Prepare random postfix
+      set_fact:
+        rpfx: "{{ 1000 | random }}"
+      run_once: yes
+
 
 - name: demo
   hosts: localhost
 # The modules of azure container registry and azure container instance are available since Ansible version 2.5.
 # But if you are using Ansible version 2.4, you need to have below 2 lines to get modules of azure container registry and azure container instance from the playbook role of azure_preview_modules.
-# roles:
-#   - Azure.azure_preview_modules
+#  roles:
+#    - Azure.azure_preview_modules
```

Comment:
seems role is not necessary

Reply:
what about 2.4?

Reply:
ansible has release. playbooks is code also, they are not supposed to be runnalble against all ansible release. we should add comments in this playbook to call out which version of ansible this playbook is run against with. This info is also aligns with sample repo CI.